contracts/modules/common/RelayerModule.sol

```diff
@@ -90,6 +91,10 @@ abstract contract RelayerModule is BaseModule {
         require(checkAndUpdateUniqueness(_wallet, _nonce, signHash), "RM: Duplicate request");
         require(verifyData(_wallet, _data), "RM: Target of _data != _wallet");
         (uint256 requiredSignatures, OwnerSignature ownerSignatureRequirement) = getRequiredSignatures(_wallet, _data);
+        if (requiredSignatures == 0) {
+            require(ownerSignatureRequirement == OwnerSignature.Anyone, "RM: Wrong number of required signatures");
+        }
+
```

Comment:
I think it would be a little cleaner to have that condition in a single line require:
```
require(requiredSignatures > 0 || ownerSignatureRequirement == OwnerSignature.Anyone, "RM: Wrong number of required signatures");
```